packages/next/src/sourcemapped.ts

```diff
@@ -99,3 +99,42 @@ export function stringifySourceMap(
   delete obj.sourcesContent;
   return JSON.stringify(obj);
 }
+
+// Based on https://github.com/thlorenz/convert-source-map/blob/f1ed815b4edacfa9c3c5552dd342e71a3cffbb0a/index.js#L4 (MIT license)
+// Groups: 1: media type, 2: MIME type, 3: charset, 4: encoding, 5: data.
+const SOURCE_MAP_COMMENT_REGEX =
+  // eslint-disable-next-line no-useless-escape
+  /^\s*?\/[\/\*][@#]\s+?sourceMappingURL=data:(((?:application|text)\/json)(?:;charset=([^;,]+?)?)?)?(?:;(base64))?,(.*?)$/gm;
+
+function isValidSourceMapData(encoding: string, data: string): boolean {
+  // Remove any spaces and the trailing `*/`.
```

Comment:
The comment says "trailing" but `.replace()` will only replace the first one.

For example, try this:

```js
'/*first*/ and /*second*/'.replace('*/', '')
```

Reply:
@nuta It might be good to add a test with a comment like that to see how it behaves now

Reply:
Good idea. Added!